### PR TITLE
BaSP-MR-22: class getall getbyid post put and delete test

### PR DESCRIPTION
**Pull request by VictoriaNegrotto**
I worked on methods: Get All, Get By Id, Post, Put and Delete tests.
I populated this methods with the ActivityId and TrainerId.

### CODE REVIEW

File: src/tests/classes.test.js [hunk 1]
```diff
@@ -1,3 +1,101 @@
...
+    expect(response.status).toEqual(404);
+    expect(response.error).toBeTruthy();
+  });
+
+  test('length of array should be greater than 1', async () => {
+    const response = await request(app).get('/api/classes').send();
+    expect(response.body.data.length).toBeGreaterThan(1);
+  });
+
+  test('should return the class Id', async () => {
```
Comment (fernandofernandezuy) [inline src/tests/classes.test.js:46]: I think the you could rename this test to make more sense with the expected response.

File: src/tests/classes.test.js [hunk 2]
```diff
@@ -1,3 +1,100 @@
...
+    const response = await request(app).post('/api/classes/').send(mockClasses);
+    expect(response.body.message).toBe('Class created!');
+    expect(response.statusCode).toBe(201);
+    expect(response.body.error).toBeFalsy();
+  });
+
+  test('Should get error 500 when you create a class', async () => {
+    const res = await request(app).post('/api/classes').send();
+    expect(res.status).toBe(500);
+    expect(res.status).toBeTruthy();
```
Comment (FontanR) [inline src/tests/classes.test.js:68]: i think here should be res.error or res.body.error instead of res.status

**Review by FontanR — APPROVED** [review]
LGTM

**Review by LucianoStradiot — APPROVED** [review]

**Review by GaloDurante — APPROVED** [review]
Good!!

**Pull request merged**